responsibleai/tests/test_rai_insights_cohort_filter.py

```diff
@@ -147,6 +150,43 @@ def test_cohort_filter_in_the_range_of(self):
                          model_task,
                          filters=filters)
 
+    @pytest.mark.parametrize('arg, correct_prediction',
+                             [([1], False), ([0], True)])
+    def test_cohort_filter_multiclass_classification_outcome(
+            self, arg, correct_prediction):
+        X_train, X_test, y_train, y_test, feature_names = create_iris_pandas()
+        model = create_sklearn_svm_classifier(X_train, y_train)
+        model_task = ModelTask.CLASSIFICATION
+        categorical_features = []
+
+        # the index 1, corresponds to incorrect prediction
+        # the index 0 correspond to correct prediction
+        filters = [{'arg': arg,
```

Comment:
nit: move the arg/column/method/includes to constants here and below

Reply:
As responded in earlier PRs, there is an internal work item for this ask.